src/components/app/App.jsx

```diff
@@ -15,10 +17,22 @@ class App extends React.Component {
     return (
       <Router>
         <React.Fragment>
```

Comment:
Why the `<Fragment>` here?

Reply:
Crash at execution without <Fragment> : "A <Router> may have only one child element"

Reply:
We should probably wrap the routes in a `Switch` component, since we want to route *exclusively*, not *inclusively*. Should fix the crash :)